lib/amqp_client.js

```diff
@@ -169,11 +169,11 @@ AMQPClient.prototype.connect = function(url) {
 };
 
 /**
- * Creates a sender link for the given address, with optional link options
+ * Creates a sender link for the given address, with optional link policy
  *
  * @method createSender
- * @param {string} [address]    An address to connect this link to
- * @param {*} [options]         Options used for creating the link
+ * @param {String} [address]    An address to connect this link to
```

Comment:
My understanding was that `[params_in_brackets]` were being marked optional, and address is non-optional.